lib/gui/os/window-progress.js

```diff
@@ -38,12 +39,16 @@ exports.currentWindow = electron.remote.getCurrentWindow();
  * Show progress inline in operating system task bar
  *
  * @param {Number} percentage - percentage
+ * @param {String} status - flashing/validating status
  *
  * @example
  * windowProgress.set(85);
  */
-exports.set = (percentage) => {
+exports.set = (percentage, status) => {
```

Comment:
Maybe we can make a separate function that's also used within the `set` function? That way, in the future, should we want to set the title to something arbitrary somewhere, we can do that.